src/sonic-ctrmgrd/ctrmgr/container_action

```diff
@@ -0,0 +1,304 @@
+        update_data(feature, {
+            CURRENT_OWNER: "none",
+            REMOTE_STATE: "none",
+            DOCKER_ID: ""})
+
+    debug_msg("END")
+
+
+def container_kill(feature):
+    debug_msg("BEGIN")
+    set_owner, _ = read_config(feature)
+    current_owner, remote_state, docker_id = read_state(feature)
+    remove_label = (set_owner != "local") or (current_owner != "local")
+
+    debug_msg("{}: set_owner:{} current_owner:{} remote_state:{} docker_id:{}".format(
+        feature, set_owner, current_owner, remote_state, docker_id))
+
+    update_data(feature, {SYSTEM_STATE: "down"})
+
+    if not docker_id:
+        docker_id = feature
+
+    if remove_label:
+        set_label(feature, False)
+
+    if docker_id:
+        docker_action("kill", docker_id)
+
+    proc = subprocess.Popen("container_state.py down -f {}".format(feature))
+    proc.wait()
+
+    debug_msg("END")
+
+
+def container_wait(feature):
+    debug_msg("BEGIN")
+
+    set_owner, fallback = read_config(feature)
+    current_owner, remote_state, docker_id = read_state(feature)
+    pend_wait_secs = 0
+
+    if not docker_id:
+        if (set_owner == "local") or (current_owner == "local"):
+            docker_id = feature
+        elif fallback:
+            pend_wait_secs = get_config_data(
+                    SONIC_CTR_CONFIG_PEND_SECS, DEFAULT_PEND_SECS)
+
+    debug_msg("{}: set_owner:{} ct_owner:{} state:{} id:{} pend={}".format(
+        feature, set_owner, current_owner, remote_state, docker_id,
+        pend_wait_secs))
+
+    while not docker_id:
+        if fallback:
+            pend_wait_secs = pend_wait_secs - WAIT_POLL_SECS
+            if pend_wait_secs < 0:
+                break
+
+        time.sleep(WAIT_POLL_SECS)
+
+        current_owner, remote_state, docker_id = read_state(feature)
+
+        debug_msg("wait_loop: {} = {} {} {}".format(feature, current_owner, remote_state, docker_id))
+
+        if (remote_state == "pending"):
+            update_data(feature, {REMOTE_STATE: "ready"})
+
+    if not docker_id:
+        # Clear remote state and exit.
+        # systemd would restart and fallback to local
+        update_data(feature, { REMOTE_STATE: "none" })
+        debug_msg("{}: Exiting to fallback as remote is *not* starting".
+                format(feature))
+    else:
+        debug_msg("END -- transitioning to docker wait")
+        docker_action("wait", docker_id)
+
+
+def container_id(feature):
+    current_owner, _, docker_id = read_state(feature)
+    if ((not docker_id) and (
+            (set_owner == "local") or (current_owner == "local"))):
+        docker_id = feature
+    print(docker_id)
+
+
+def main():
+    parser=argparse.ArgumentParser(description="container commands for start/stop/wait/kill/id")
+    parser.add_argument("action", choices=["start", "stop", "wait", "kill", "id"])
+    parser.add_argument("name")
+
+    args = parser.parse_args()
+
+    init()
+
+    if args.name not in remote_features:
```

Comment:
Instead of checking hardcoded remote_features, can we check stored config value?